app/models/ticket_comment.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class TicketComment < ApplicationRecord
+  belongs_to :ticket
+  belongs_to :acting_user, class_name: 'User', foreign_key: :acting_user_id
```

Comment:
Bit surprised that the `foreign_key` is necessary. Usually Rails assumes `#{name_of_link}_id` as default. Have you tried without it?